ratdb/CHANNEL_STATUS_DEFAULTS.ratdb

```diff
@@ -12,3 +12,9 @@
   "run_range": [0, 0],
   "default_value": 1,
 }
+{
```

Comment:
Can you add a new line here to separate the tables? I think it would make it slightly easier to read.

Reply:
done